WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -362,6 +363,23 @@ public void onStart() {
         EventBus.getDefault().registerSticky(this);
     }
 
+    // Removes app notifications from the system bar
+    class CancelNotificationsTask extends AsyncTask<Void, Void, Void> {
```

Comment:
> instead of using an AsyncTask this could simply be run inside a thread

You're right, AsyncTasks should be used if you're updating the UI at the end. I changed it to a thread in https://github.com/wordpress-mobile/WordPress-Android/commit/d71b433d906e5457831bb3d17bf1c257ac884a73